src/components/PopupWithFormSubmit.js

```diff
@@ -0,0 +1,15 @@
+import Popup from "./Popup.js";
+export default class PopupWithFormSubmit extends Popup {
```

Comment:
`PopupWithFormSubmit` should be named `PopupWithConfirmation` according to the check list